domains/xos.json

```diff
@@ -4,7 +4,6 @@
         "email": "pollardjackson@outlook.com"
     },
     "record": {
-        "CNAME": "acc468f2-4bc9-4dfa-8059-bc9edf2eb513.id.repl.co",
-        "TXT": "replit-verify=acc468f2-4bc9-4dfa-8059-bc9edf2eb513"
+        "CNAME": "acc468f2-4bc9-4dfa-8059-bc9edf2eb513.id.repl.co"
```

Comment:
Unable to access CNAME record.

Reply:
this is because i currently have the original site in development this is when it is active: 
![image](https://github.com/is-a-dev/register/assets/73853537/8b409d19-57f8-456b-babb-04caa3a0bf02)

Reply:
i made changes to the file

Reply:
What do you mean invalid file

Reply:
@DEV-DIBSTER Please explain why it is an invalid file

Reply:
I have updated the labels and make further requests that you may address.